.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+  "python.pythonPath": "zulip-api-py3-venv/bin/python3.7"
+}
```

Comment:
Probably this should go in `.gitignore`; the Python version at least isn't fixed.

Reply:
In my last commit, the vs code settings file is ignored